proposals/0024/GOVERNANCE.md

```diff
@@ -0,0 +1,30 @@
+# Governance
+
+This document describes how the Liberal contribution governance structure is implemented in the Tinkerbell community.
+
+## What is Liberal contribution?
+
+Under a liberal contribution model, the people who do the most work [in the prescribed way] are recognized as most influential, but this is based on current work and not historic contributions.
+Major project decisions are made based on a consensus seeking process (discuss major grievances) rather than pure vote, and strive to include as many community perspectives as possible.
+Popular examples of projects that use a liberal contribution model include Node.js and Rust <sup>[1]</sup>.
+
+## Community Roles
+
+This doc outlines the various roles in the Tinkerbell community.
+
+One of the overarching goals of having different roles in the community is to make sure to "allow good changes to enter the code with a minimum of fuss"<sup>[2]</sup>.
+The following criteria is to be used when considering and individual for a role.
+
+- a pattern of good judgement in regard to the code bases and to oneself
+- demonstrated history of "playing nice" in the community
+- amount and consistency of contributions and activity
+
+| Role        | Responsibilities | Requirements | Defined by |
+| -----       | ---------------- | ------------ | ------- |
+| Contributor | Submit contributions | signed DCO | n/a |
+| Member      | issue and discussion administration, PR approval | history of activity, good judgement, and contribution in the community and the repositories, Sponsored by 2 Committers | Tinkerbell GitHub org member |
+| Committer   | Right to make changes to the code bases | same as Member and shows leadership in the community and the repositories, sponsored by 2 Maintainers | [OWNERS] file reviewer entry |
+| Maintainer  | Voting privileges | same as Committer, voted in by existing Maintainers | [OWNERS] file reviewer entry |
```

Comment:
Is the difference between committer and maintainer only that maintainers get to vote on committers? I'm not sure we need to have a separation between committer and maintainer if we're seeking to come to consensus vs votes except for maintainer status votes. I'd rather just see the role of committer renamed to maintainer (to avoid the optics of a committer able to push commits w/o review/feedback for e.g.)

Reply:
[consensus seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making) will fall back to a vote if necessary so the voting is more than just for voting in new maintainers.

Reply:
gotcha, thanks for the update